frontends/ol-util/src/hooks/index.ts

```diff
@@ -1,2 +1 @@
 export { default as useSearchParams } from "./useSearchParams"
-export { default as useId } from "./useId"
```

Comment:
I deleted this hook because I found a better way to do what I was after.

Previously I was using this for
```tsx
<h2 id={elementId}>Fields of Study</h2>
<ul aria-labelledby={elementId} className="field-list">
```

Now those two elements are both children of a `<section />`, which seems like a more semantic way to associate the title with the list.